app/views/admin/products_v3/_table.html.haml

```diff
@@ -21,8 +21,8 @@
       %col.col-sku{ width:"8%", style:"min-width: 6em" }
       %col.col-unit_scale{ width:"8%" }
       %col.col-unit{ width:"8%" }
-      %col.col-price{ width:"5%", style:"min-width: 5em" }
-      %col.col-on_hand{ width:"10%"}
+      %col.col-price{ width:"10%", style:"min-width: 5em" }
```

Comment:
shouldn't we increase `min-width` to a sensible value so that most prices are displayed fully ?

Reply:
Maybe, but it could have a consequence affecting other columns on a small screen. I forget, but I think 5em covers most prices under 100.00 (which hopefully covers most products in most countries..)